addons/fluxcd/parameter.cue

```diff
@@ -1,10 +1,10 @@
 parameter: {
 	//+usage=Base URL for images that the FluxCD controllers use, e.g. ghcr.io
-	registry?: *"" | string
```

Comment:
Hi @FogDong . I think maybe we should remain this `?` character.
This inputBox is required when we remove `?` character. But I actually don't want to input anything just use the default DockerHub Registry is fine.
<img width="386" alt="image" src="https://user-images.githubusercontent.com/53958238/193447101-f21b8c15-6b74-43e4-957c-264b4869f966.png">